tests/Doctrine/Tests/DBAL/Types/DateImmutableTypeTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\DateImmutableType;
+use Doctrine\DBAL\Types\Type;
+
+class DateImmutableTypeTest extends \PHPUnit_Framework_TestCase
```

Comment:
Since when are we doing that? Never used it before. Please give an example here if I should.